pdns/dnsname.cc

```diff
@@ -53,9 +51,9 @@ void DNSName::packetParser(const char* pos, int len, int offset, bool uncompress
       int newpos = (labellen << 8) + *(const unsigned char*)pos;
 
       if(newpos < offset) {
-        if (++d_recurse > 100)
+        if (++depth > 100)
           throw std::range_error("Abort label decompression after 100 redirects");
-        packetParser(opos, len, newpos, true);
+        packetParser(opos, len, newpos, true, 0, 0, 0, depth);
```

Comment:
That functionality is unused in pdns auth, I'm guessing it's unused anywhere. Let's not hold this PR on the question.